pkg/apis/bundle/v1alpha1/cluster_bundle.proto

```diff
@@ -42,6 +41,9 @@ message ClusterBundle {
 
 // A spec object that wraps the Cluster Bundle.
 message ClusterBundleSpec {
+  // Name of the cluster bundle.
+  string name = 1;
```

Comment:
Is my understanding that the "name" string here is intended to identify a collection of bundles, rather than a specific bundle?

I'm worried that overloading "name" is going to be confusing, because most will assume ObjectMeta.Name when "name" is used in conversation - and in that context "name" always maps to a singular object per namespace per kind. 

Some possible other options:
- class
  - "Class" in K8s is often used to describe classes of resources that can be scheduled: StorageClass, ResourceClass, RuntimeClass.
  -  Since Bundles could plausibly represent the flavors of clusters one is allowed to create (and maybe schedule to), maybe Bundle<=>ClusterClass is true if you squint at it. But I kind of had to squint to make this one work.
- category (casually used in K8s... shows up a near taints a lot when you grep for "category", but not a very common term)
- type (most common K8s use seems to be for sum types)
- kind (this *is* the word that K8s uses to describe config types. but like "name" it is has a specific, common meaning, and could be confusing in conversation)

Aaaargh none of these seem very good!

Reply:
Ok, so we can differentiate that the metadata.name is the resource name in a particular k8s api server that is really a reference back to some canonical "bundle" that is identified by this 'name' and a version. Maybe just "bundle name"?

Reply:
I think a description of the concrete use-case where this is necessary would help this discussion.